lib/search/search_page.dart

```diff
@@ -32,9 +41,39 @@ class SearchView extends StatelessWidget {
           child: FlexSearchBar(
             autoFocus: true,
             fromTab: fromTab,
+            onChanged: (value) {
+              BlocProvider.of<ProductSearchCubit>(context)
+                  .searchProductsAutocomplete(query: value);
+            },
           ),
         ),
       ),
+      body: BlocBuilder<ProductSearchCubit, ProductSearchState>(
+        builder: (_, state) {
+          final children = <Widget>[
+            if (state.status == Status.pending && state.products.isEmpty)
+              const Center(child: CircularProgressIndicator()),
+            for (var p in state.products) ...[
+              Container(
```

Comment:
`              Divider(
                height: 1,
                color: Colors.black.withOpacity(0.1),
              ),`

Reply:
oooooh that's better 👌